lib/text/vtt_text_parser.js

```diff
@@ -72,8 +84,8 @@ shaka.text.VttTextParser = class {
     // HLS content, which should use X-TIMESTAMP-MAP and periodStart instead.
     let offset = time.vttOffset;
 
-    // Only use 'X-TIMESTAMP-MAP' in sequence mode, as that is currently
-    // shorthand for HLS.  Note that an offset based on the first video
+    // Only use 'X-TIMESTAMP-MAP' in sequence mode or HLS.
```

Comment:
I don't think this comment is accurate. It makes it sound like we are using `X-TIMESTAMP-MAP` in all HLS, but in reality we still don't parse the tag in segment mode HLS.
We are merely taking the presence of an `X-TIMESTAMP-MAP` tag as a signal that the offset is to be calculated differently in segment mode HLS, correct?
It might be better to have a short explanation inside the new clause below, and just remove the mention of HLS from this comment entirely.

Reply:
Changed!